core/templates/dev/head/domain/url/UrlInterpolationService.js

```diff
@@ -0,0 +1,54 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
```

Comment:
What is the definition of the domain/url directory? How would I figure out whether something goes there or, say, in domain/utilities?

(Asking because I would probably have put this in domain/utilities originally.)